src/js/index.js

```diff
@@ -104,3 +108,38 @@ window.addEventListener('load', () => {
 
 	cascade.call(window, true);
 });
+
+const preference = localStorage.getItem('color-preference');
+if(preference=='dark')
+{
+	container[0].classList.add('dark');
```

Comment:
You can add this class directly to the body tag instead of the container. (`document.body.classList`)